gulpfile.js

```diff
@@ -82,8 +82,8 @@ const defaultTask = gulp.series(
   compileJavascript,
   minifyImages,
   font,
+  svgRename,
```

Comment:
When running `yarn run dev`, it will actually hang at `watchFiles` (it's watching files...) and  `svgRename` will never be executed. However, it is necessary to run it in order to add new svgs, so I took the liberty to move it up!